fe2-android/docs/README.md

```diff
@@ -102,25 +102,20 @@ are **always** the source of truth and are more up to date than the Google Doc.
 The application follows the Model-View-ViewModel (MVVM) pattern and uses
 this [guide to app architecture](https://developer.android.com/jetpack/guide).
 
-- The **View** consists of all the activities and fragments of the application. In this project
-  there are only two activities respectively representing the detail and home view, this might
-  change because one activity should represent one functionality of the application. The fragments
-  represent a part of the UI within one of those activities.
+- The **View** consists of all the activities and fragments of the application. They are responsible for the displaying part.
 - The **ViewModel** implements the UI logic and prepares and manages the data used by the activities
-  and fragments. In this project each activity has its own view model.
-- The **Model** consists of the local and remote data source, the model classes and the repository.
-  The repository eases the data retrieval for the rest of the application. It is a mediator between
-  the view model and the different data sources. In this application there are two data sources, the
-  database and the PoP backend.
-  The [Room](https://developer.android.com/reference/androidx/room/package-summary) persistence
-  library is used to define the application's database, the entities and Data Access Object.
+  and fragments. Here, there's roughly a view model per feature. A fragment may use more than 1 view model.
+- The **Model** consists of the local and remote data source, the model classes and the repositories.
```

Comment:
```suggestion
- The **Model** consists of the model classes stored in the repositories.
```
Actually we should probably just dump the whole principle of remote vs local datasource

There is a whole package that is not used anywhere and it might only confuse people.